test/src/main/java/org/apache/accumulo/test/start/KeywordStartIT.java

```diff
@@ -113,6 +115,8 @@ public void testExpectedClasses() {
     expectSet.put("check-server-config", CheckServerConfig.class);
     expectSet.put("compaction-coordinator", CoordinatorExecutable.class);
     expectSet.put("compactor", CompactorExecutable.class);
+    expectSet.put("config-print", ConfigPropertyPrinter.class);
+    expectSet.put("config-upgrade", ConfigPropertyUpgrader.class);
```

Comment:
It is a one-off, and as soon as we can assert that 2.1 is the base version for any future upgrades, all of the upgrade code related to the property conversion should be removed - that includes this utility.  If other general, pre-start utilities are needed, they can be created at that time.  

I have no issue with renaming if that makes it clearer when / if it should be run and if it establishes a convention if future tasks are needed - just seeking clarification on your intent with this suggestion.

Reply:
In general, I expect KeywordExecutables to last for more than one version. These are very user-facing, and users start to rely on these as entry points into utilities. If it's going to be deleted in subsequent versions, I'd rather avoid the KeywordExecutable, and just instruct users to execute it as a regular `main` class.
However, my intent with this comment was that we have the opportunity to establish a convention if it's needed in future. In versions where it isn't needed, executing it can be a NOOP, but we could still keep the KeywordExecutable around.

Basically, I think the choice is: no keyword and delete it OR establish a convention with a keyword and keep it around

Reply:
It seems there's precedent for this, with the convert config one, so I don't care as much now. :smiley_cat: